src/DynamoCoreWpf/Views/Core/NodeView.xaml.cs

```diff
@@ -219,8 +219,11 @@ private void OnNodeViewLoaded(object sender, RoutedEventArgs e)
         private void NodeModel_ConnectorAdded(Graph.Connectors.ConnectorModel obj)
         {
             // If the mouse does not leave the node after the connnector is added,
-            // try to show the preview bubble without new mouse enter event.   
-            Dispatcher.BeginInvoke(new Action(TryShowPreviewBubbles), DispatcherPriority.Loaded);
+            // try to show the preview bubble without new mouse enter event. 
+            if (IsMouseOver)
```

Comment:
This looks OK. Still, I feel there is an event that was not flushed. when you add the connector, the events should get flushed  out immediately.. then when you mouse leave on the ports, then enter the node, it should kick off the mouse enter event on the node. Merge this in. I don't think this will hit any performance issues.,
Thoughts @mjkkirschner ?